src/storage/txn/scheduler.rs

```diff
@@ -41,7 +41,7 @@ use storage::mvcc::{MvccTxn, MvccReader, Error as MvccError};
 use storage::{Key, Value, KvPair};
 use std::collections::HashMap;
 use mio::{self, EventLoop};
-use util::transport::SendCh;
+use util::transport::{SendCh, MAX_SEND_RETRY_CNT};
```

Comment:
MAX_SEND_RETRY_CNT is a global constant and used everywhere when calling `try_send()`. Maybe it's good to  make retry count a attribute of this channel, and ignore it when calling `try_send()`.